### DISCUSSION TITLE
Annotation of Macaca mulatta for multiome analysis

### DISCUSSION THREAD

**Discussion by sylestiel** [stuart-lab/signac#1534] [General]
Hi Tim,

I've been trying to annotate the Rhesus Macaque genome as follows. However, I'm not able to run the analysis. It gives me an annotation error. Can you see if the following is correct:

gtf <- rtracklayer::import('/Volumes/BKUP2/rheMac10_LBN/Macaca_mulatta.Mmul_10.110.gtf')
gene.coords <- gtf[gtf$type == 'gene']
seqlevelsStyle(gene.coords) <- "Ensembl"
annotation <- gene.coords
seqlevels(annotation) <- paste0('chr', seqlevels(annotation))

**Comment by timoast** [comment]
Hi @sylestiel, subsetting to only type=="gene" may cause some issues, I recently updated our FAQ to change this guidance since some others were having similar issues too.

**Reply by sylestiel** [reply]
@timoast 

I used this instead:

annotations <- GRanges(seqinfo(BSgenome.Mmulatta.UCSC.rheMac10))
seqlevelsStyle(annotations) <- 'Ensembl'
genome(annotations) <- "Mmul_10"
 ### **But ran into the following error:**
integrated <- RegionStats(integrated, genome = BSgenome.Mmulatta.UCSC.rheMac10)
**Warning in RegionStats.default(object = regions, genome = genome, verbose = verbose,  :
  Not all seqlevels present in supplied genome**

Can you suggest ways to address this issue.


### Alternatively, if I were to use
library(ensembldb)
gtf <- "/Users/pediatrics/Downloads/Macaca_mulatta.Mmul_10.110.gtf"
DB <- ensDbFromGtf(gtf) 
EDB <- EnsDb(DB) 
annotation <- GetGRangesFromEnsDb(ensdb = EDB) 

what would the genome when running RegionStats command. I'm confused as to how to proceed with regard to specification of genome when running RegionStats and also when it comes to Motif Analysis. Can you please clarify. Thank you in advance!

**Reply by timoast** [reply]
You need to look at how the chromosomes are named in your data and make sure they are named the same way in the genome and gene annotations you are using
